toolchain/binutils/Config.in

```diff
@@ -17,6 +17,10 @@ choice
 	config BINUTILS_USE_VERSION_2_43
 		bool "Binutils 2.43.1"
 		select BINUTILS_VERSION_2_43
+		
```

Comment:
This line contains unnecessary trailing whitespace

Reply:
thanks, fixed